source/common/runtime/runtime_features.cc

```diff
@@ -47,6 +47,11 @@ constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
     "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
+    "envoy.deprecated_features.config_source.proto:UNSUPPORTED_REST_LEGACY",
+    "envoy.deprecated_features.ext_authz.proto:use_alpha",
+    "envoy.deprecated_features.route.proto:enabled",
+    "envoy.deprecated_features.fault.proto:type",
+    "envoy.deprecated_features.route.proto:runtime_key",
```

Comment:
Nit: this could be sorted alphabetically.

Reply:
I think we could locally, but tdb if we want to have these grouped by release rather than alphabetical overall.  I'm going to merge without fixing just because I want it have a day or two where they're flipped and I'm around for questions and we can sort out ordering next release or in a follow-up?